extensions/functions_arithmetic.yaml

```diff
@@ -643,6 +643,34 @@ scalar_functions:
           on_domain_error:
             values: [ NAN, ERROR ]
         return: fp64
+  -
+    name: "radians"
+    description: >
+      Converts angle `x` in degrees to radians.
+
+    impls:
+      - args:
+          - name: x
+            value: fp32
+        return: fp64
```

Comment:
Most extension functions avoid this kind of implicit upcasting -- is there a particular reason we want this to return `fp64` for a `fp32` input?

Reply:
oh yeah...we shouldn't be implicit upcasting here.  I'll update it!  I'll also add the rounding options.